Dockerfile

```diff
@@ -19,40 +19,43 @@ ENV         LANG="en_US.UTF-8" \
             WARN_ON_STOP="true" \
             ARK_TOOLS_VERSION="${ARK_TOOLS_VERSION}" \
             ARK_SERVER_VOLUME="/app" \
+            TEMPLATE_DIRECTORY="/conf.d" \
             GAME_CLIENT_PORT="7777" \
             UDP_SOCKET_PORT="7778" \
             RCON_PORT="27020" \
             SERVER_LIST_PORT="27015" \
             STEAM_USER="steam" \
             STEAM_GROUP="steam" \
             STEAM_UID="1000" \
-            STEAM_GID="1000" \
-            STEAM_HOME="/home/steam"
+            STEAM_GID="1000"
+
+ENV         ARK_TOOLS_DIR="${ARK_SERVER_VOLUME}/arkmanager" \
+            STEAM_HOME="/home/${STEAM_USER}"
 
 RUN         set -x && \
+            dpkg --add-architecture i386 && \
             apt-get -qq update && apt-get -qq upgrade && \
-            apt-get -qq install curl lib32gcc1 lsof perl-modules libc6-i386 bzip2 bash-completion locales sudo cron && \
+            apt-get -qq install libsdl2-2.0-0:i386 libcurl4 curl lib32gcc1 lsof perl-modules libc6-i386 bzip2 bash-completion locales sudo cron && \
             sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && locale-gen && \
             addgroup --gid ${STEAM_GID} ${STEAM_USER} && \
             adduser --home ${STEAM_HOME} --uid ${STEAM_UID} --gid ${STEAM_GID} --disabled-login --shell /bin/bash --gecos "" ${STEAM_USER} && \
             echo "${STEAM_USER} ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
             usermod -a -G sudo ${STEAM_USER} && \
             mkdir -p ${ARK_SERVER_VOLUME} ${STEAM_HOME}/steamcmd && \
-            curl -L https://github.com/FezVrasta/ark-server-tools/archive/v${ARK_TOOLS_VERSION}.tar.gz \
+            curl -L "https://github.com/FezVrasta/ark-server-tools/archive/v${ARK_TOOLS_VERSION}.tar.gz" \
```

Comment:
Maybe change the URL to the new repo/group?
https://github.com/arkmanager/ark-server-tools instead of FezVrasta